tensorboard/webapp/types/api.ts

```diff
@@ -72,6 +72,12 @@ export interface Environment {
   data_location: string;
   /** @export */
   window_title: string;
+  /** @export */
+  experiment_name?: string;
+  /** @export */
+  experiment_description?: string;
+  /** @export */
+  creation_time?: number;
```

Comment:
Does it make sense to also include `update_time` at this point?

Reply:
Also, is it common to include a units here?  I see the time is a number but is it seconds since epoch?  ms?  Is there a standard practice here?

Reply:
> Does it make sense to also include update_time at this point?

TensorBoard.dev does not yet retrieve update_time so I don't think I should add it.

> Also, is it common to include a units here? I see the time is a number but is it seconds since epoch? ms? Is there a standard practice here?

Unfortunately this is out of my control. This declaration just mimics existing behavior. I'm not sure where these fields are originally declared (if anywhere) but this is where they are set:

https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/plugins/core/core_plugin.py;l=149;drc=fbdb2b7b674538ff86c4704dcd3f1f0ddc25737a